test/e2e/testing-manifests/storage-csi/gce-pd/csi-controller-rbac.yaml

```diff
@@ -39,11 +48,12 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: csi-gce-pd-controller-sa
+    namespace: default
```

Comment:
Is this right? I thought the csi driver gets deployed in the e2e test namespace?

Reply:
removed namespace, namespace will be patched before deployment